JustSaying.IntegrationTests/AwsTools/SqsQueueIntegrationTests.cs

```diff
@@ -17,12 +17,12 @@ protected override SqsQueueByName CreateSystemUnderTest()
         {
             QueueUniqueKey = "test" + DateTime.Now.Ticks;
             var queue = new SqsQueueByName(RegionEndpoint.EUWest1, QueueUniqueKey, CreateMeABus.DefaultClientFactory().GetSqsClient(RegionEndpoint.EUWest1), 1, new LoggerFactory());
-            queue.Exists();
+            queue.ExistsAsync().GetAwaiter().GetResult();
```

Comment:
I didn't do this cause the base class is in JustBehave, however I could always create a new base class in this project, I'll take a look 👍